documents/Specification/MaterialX.Specification.md

```diff
@@ -1148,19 +1123,9 @@ Applications may also reference other renderer-specific named spaces, at the exp
 
 Global nodes generate color data using non-local geometric context, requiring access to geometric features beyond the surface point being processed.  This non-local context can be provided by tracing rays into the scene, rasterizing scene geometry, or any other appropriate method.
 
-```xml
-  <ambientocclusion name="occl1" type="float">
-    <input name="maxdistance" type="float" value="10000.0"/>
-  </ambientocclusion>
-```
-
 Standard Global nodes:
 
-<a id="node-ambientocclusion"> </a>
-
-* **`ambientocclusion`**: Compute the ambient occlusion at the current surface point, returning a scalar value between 0 and 1.  Ambient occlusion represents the accessibility of each surface point to ambient lighting, with larger values representing greater accessibility to light.  This node must be of type float.
-    * `coneangle` (float): the half-angle of a cone about the surface normal, within which geometric surface features are considered as potential occluders.  The unit for this input is degrees, and its default value is 90.0 (full hemisphere).
-    * `maxdistance` (float): the maximum distance from the surface point at which geometric surface features are considered as potential occluders.  Defaults to 1e38, e.g. "unlimited".
+There are currently no Global Nodes implemented in MaterialX.
```

Comment:
Since our sole example of a Global Node has been moved to the Proposals document, perhaps we should take this opportunity to remove the section itself, and to consider `ambientocclusion` to be a proposal for the NPR Nodes specification?

Reply:
There are certainly use cases for ambient occlusion in pattern generation in a PBR render, e.g. for procedural dirt accumulation in crevices, but if moving this node over to the NPR document would make it easier or more likely for it to actually get implemented, I'm all for it.
I'm curious what others think.

Reply:
Good points, @dbsmythe, and I'm open to keeping the current Global Nodes category for this proposal.  Maybe we should delete the empty section from the main specification, though, making it a category that will come into existence when `ambientocclusion` or a similar node gets added?

Reply:
+1 for removing the empty section, I think that when we get an Implementation of `ambientocclusion` that will be a good time to debate what section it belongs to, because we will perhaps have a little more context around that work.

For instance, in the OSL TSC, we sometimes debate the difference between ambient occlusion that is resolved at shade time, perhaps even implemented inside the shader, vs a deferred ambient occlusion accessed via a closure, all that to be said, the devil is in the details, and seeing the future is sometimes hard :)

Reply:
The empty Globals section (as well as any references to it) have been removed in a new commit.